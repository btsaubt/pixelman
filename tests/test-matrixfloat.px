def int main(){
	float[5][5] m1;
	int i;
	int j;
	m1 = [1.0,2.0,3.0,4.0,5.0]|[1.0,2.0,3.0,4.0,5.0]|[1.0,2.0,3.0,4.0,5.0]|[1.0,2.0,3.0,4.0,5.0]|[1.0,2.0,3.0,4.0,5.0];
	for(i = 0;i<5;i=i+1){
		for(j=0;j<5;j=j+1){
			print(m1[i][j]);
		}
	}
}