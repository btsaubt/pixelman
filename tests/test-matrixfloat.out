1.0000002.0000003.0000004.0000005.0000001.0000002.0000003.0000004.0000005.0000001.0000002.0000003.0000004.0000005.0000001.0000002.0000003.0000004.0000005.0000001.0000002.0000003.0000004.0000005.000000